index.html

```diff
@@ -1,29 +1,106 @@
+        <p class="decription__text">Generowany dokument zostanie utworzony jako plik .docs, możliwy do edycji w Wordzie
+            i ewentualnego
+            dopisania dodatkowych informacji.</p>
+        <img src="assets/images/description-graphics.svg" alt="" class="description__graphics">
+    </section>
+    <section class="features">
+        <h3 class="titles features__title">Zobacz Alinkę w akcji</h3>
+        <div class="features__box">demo</div>
+        <div class="features__list">
+            <div class="feature features__item">
+                <img src="assets/images/feature-free.svg" alt="" class="feature__icon">
+                <p class="feature__text">Za darmo</p>
+            </div>
+            <div class="feature features__item">
+                <img src="assets/images/feature-easy.svg" alt="" class="feature__icon">
+                <p class="feature__text">Łatwa w osłudze</p>
+            </div>
+            <div class="feature features__item">
+                <img src="assets/images/feature-safe.svg" alt="" class="feature__icon">
+                <p class="feature__text">Bezpieczna</p>
+            </div>
+        </div>
+    </section>
+    <section class="download">
+        <h3 class="titles download__title">Zacznij pracować z Alinką!</h3>
+        <div class="download__buttons">
+            <button class="button button--color">Pobierz</button>
+            <button class="button button--regular">Zmień wersję</button>
+        </div>
+        <img src="assets/images/download-graphics.svg" alt="" class="download__graphics">
+    </section>
+    <footer class="footer">
+        <div class="icons footer__icons">
+            <img src="assets/images/Code_for_Poznan_final-01.png" alt="Code for Poznan logo" class="icons__logo">
+            <p class="icons__email">hello@codeforpoznan.pl</p>
+            <div class="social-media icons__social-media">
+                <a href="https://www.facebook.com/CodeForPL/" class="social-media__item">
+                    <img src="assets/images/facebook.svg" alt="Facebook">
+                </a>
+                <a href="https://pl.linkedin.com/company/codeforpoznan" class="social-media__item">
+                    <img src="assets/images/linked-in.svg" alt="linked-in">
+                </a>
+                <a href="https://github.com/CodeForPoznan" class="social-media__item">
+                    <img src="assets/images/github.svg" alt="Github-link">
+                </a>
+            </div>
+        </div>
+        <div class="footer__author">
+            <p class="author__text">Created by <b>Code for Poznan</b></p>
+        </div>
+        <nav class="nav footer__nav">
+            <ul class="nav__list">
+                <li class="nav__item"><a href="https://codeforpoznan.pl/" class="nav__link">O nas</a></li>
+                <li class="nav__item"><a href="#" class="nav__link">Pomóż Alince</a></li>
+            </ul>
+        </nav>
+    </footer>
 </body>
-</html>
 
+</html>
```

Comment:
don't leave empty spaces, and something is wrong with nesting head and body, use prettier or other extension for formatting code